commands/get/get.go

```diff
@@ -15,6 +16,7 @@ func NewGetCmd(in io.Reader, out io.Writer, store storage.Storer) cli.Command {
 		Subcommands: []cli.Command{
 			NewProjectsCmd(in, out, store),
 			NewProjectCmd(in, out, store),
+			templates.NewGetTemplateCmd(in, out, store),
```

Comment:
hmm. So we have a sub package of get. Ahh I think I know why, this is so you can have a file named project.go ? I think it might make sense to have it as get/templates.go ? Not sure though will take a look through rest of PR